packages/mui-styles/src/makeStyles/makeStyles.test.js

```diff
@@ -531,24 +558,26 @@ describe('makeStyles', () => {
           <StressTest />
         </StylesProvider>,
       );
+
       expect(sheetsRegistry.registry.length).to.equal(2);
       expect(sheetsRegistry.toString()).to.equal(`.makeStyles-root-2 {
   color: white;
   background-color: black;
 }`);
 
-      act(() => {
```

Comment:
`fireEvent` already handles `act` for us.

Reply:
Should we remove it then?